src/RobotController.cpp

```diff
@@ -59,8 +59,7 @@ Eigen::VectorXd RobotController::computeControlTorques() {
 		control_torques += task->computeTorques() - previous_tasks_disturbance;
 	}
 	previous_tasks_disturbance =
-		(MatrixXd::Identity(dof, dof) -
-		 _redundancy_completion_task->getTaskNullspace().transpose()) *
+		_redundancy_completion_task->getPreviousTasksNullspace().transpose() *
```

Comment:
I don't think this is correct. This is N_prec, but what we want here is the range space of this specific task